src/components/moj-timeline/README.md

```diff
@@ -3,7 +3,25 @@
 - [Guidance](https://mojdt-design-system.herokuapp.com/components/timeline)
 - [Preview](https://mojdt-frontend.herokuapp.com/components/timeline)
 
+## Dependencies
+
+- [Moment.js](https://momentjs.com/)
+
+### Installation
```

Comment:
This is not specific to timeline, these filters are available at the package level and so should be provided in the main readme/installation guides if they exist. They will in the design system.

Reply:
I have removed the line mentioning `moment.js` but will leave the installation instructions in lieu of another place to put the information.